packages/website/app/routes/home.tsx

```diff
@@ -1,36 +1,101 @@
-				<h1 w-text="60px" >探索IOClub首页</h1>
-				<p>MemberList</p>
-				<main>
-					<Outlet />
-				</main>
-			</div>
-			<div className="bottom">
-			</div>
-		</div>
-	)
-}
\ No newline at end of file
+import type {
+	LoaderFunction,
+	MetaFunction,
+} from '@remix-run/server-runtime'
+import type { IArticleBox } from '~/types/articleBox'
+
+import { Link , useFetcher } from '@remix-run/react'
+import { json } from '@remix-run/server-runtime'
+import {useEffect, useState } from 'react'
+
+import { ArticleBox } from '~/components/articleBox/articleBox'
+import { db } from '~/utils/db.server'
+
+export const meta: MetaFunction = () => ({
+	title: 'Home',
+})
+
+export const handle = {
+	breadcrumb: () => <Link to="/home">Home</Link>,
+}
+export const loader: LoaderFunction = async ({ request }) => {
+	const index =
+    parseInt(new URL(request.url).searchParams.get('index') as string) || 0
+	const data = await getData(index)
+	return json(data)
+}
+const getData = async (articleBoxId = 1) => {
+	const data: IArticleBox = await db.articleBox.findUnique({
+		where: {
+			id: articleBoxId,
+		},
+		select: {
+			id: true,
+			type:true,
+			name: true,
+			title: true,
+			content: true,
+			createTime: true,
+		},
+	})
+	return data
+}
+
+
+export default function Home() {
+	const fetcher = useFetcher()
+	const [index, setIndex] = useState(0)
+	const [articleBoxList, setArticleBoxList] = useState<IArticleBox[]>([])
+	const [newData, setNewData] = useState({})
+	const [end,setEnd] = useState(false)
+	useEffect(() => {
+		fetcher.load(`/home?index=${index}`)
+	}, [index])
+	useEffect(() => {
+		if (fetcher.data == undefined) {
+			setEnd(true)
+			return
+		}
+		setEnd(false)
+		setNewData(fetcher.data)
+		const newArticleBoxList = articleBoxList
+		newArticleBoxList.push(fetcher.data)
+		setArticleBoxList(newArticleBoxList)
+	}, [fetcher.data])
+	useEffect(() => {
+		intersectionObserverFunc()
+	}, [newData])
+	const intersectionObserverFunc = () => {
+		let newIndex = index
+		const articleBoxListSentry = document.querySelector('.article-box-list-sentry') as Element
+		const intersectionObserver = new IntersectionObserver(function (entries) {
+			if (entries[0].intersectionRatio > 0) {
+				newIndex += 1
+				setIndex(newIndex)
+				intersectionObserver.unobserve(articleBoxListSentry)
+			}
+		})
+		intersectionObserver.observe(articleBoxListSentry)
+	}
+
+	return (
+		<div className="home-container w-screen  min-h-screen">
+			<div className="top h-3xl w-100vw mb-20 home-top-img ">
+			</div>
+			<div className="mx-auto mb-300px" w-w="1024px">
+				<h1 w-text="60px">探索IOClub首页</h1>
+				<p>MemberList</p>
+				<div>
+					{articleBoxList &&
+            articleBoxList.map(articleBox=> articleBox&&<ArticleBox key={articleBox.id} data={articleBox}/>)}
+				</div>
+			</div>
+			<div
+				className="bottom article-box-list-sentry"
+				w-h="300px"
+				w-text="black center"
+				w-align="middle"
+			>{end?'END':''}</div>

```

Comment:
use `{end ? <div> : ''}`, that is wrapping `<div>` inside it. We can render one less element in DOM.

Reply:
fiexd